repl/repl.js

```diff
@@ -99,7 +106,7 @@ while (true) {
         var line = 0;
         //print the accumulated code with a ">" before the broken line.
         //Don't add the last statement to the accumulated code
-        print(lines.map(function (str) {
+        print(origLines.map(function (str) {
             return ++line + (index === line ? ">" : " ") + ": " + str;
```

Comment:
The line number will still be the line number in the code with those extra lines so now the error will be reported in the wrong place.

Reply:
We're on a code path where the code provided by the user is guaranteed to be a single line.

After we munge the code, the user code is still guaranteed to be on line 1.